### DISCUSSION TITLE
Prior Distributions

### DISCUSSION THREAD

**Discussion by diayyash** [google/meridian#170] [Q&A] [answered]
Hello, is it possible to apply Beta or Gamma distributions as priors ? We seem to get an error when we try to do this.

**Comment by vivekbeniwal** [comment, marked as answer]
Hi diayyash,

We have pushed a fix for this recently https://github.com/google/meridian/pull/157

Can you please sync with latest head and share the errors you are getting.

Regards
Vivek

**Reply by diayyash** [reply]
Hi,

We've synced the latest head and it worked well, we're no longer getting any errors. Thanks for your help!

Regards,
Dima
